documentation/docs/plugins/deploy.md

```diff
@@ -0,0 +1,40 @@
+---
+displayed_sidebar: pluginsSidebar
+---
+
+# Deploying your Plugin
+
+As previously mentioned, plugins must be hosted on your own infrastructure. This gives you full control over the technologies you use and when to update your plugin(s).
+
+## Where to Deploy
+
+You can deploy your plugin/app anywhere. But it's import to make the plugin accessible publicly from the web. Otherwise, the features of your plugin will not be visible to users.
```

Comment:
```suggestion
You can deploy your plugin/app anywhere. But it's important to make the plugin accessible publicly from the web. Otherwise, the features of your plugin will not be visible to users.
```